dist/angular-dashboard-framework.js

```diff
@@ -567,12 +567,17 @@ angular.module('adf')
           $scope.editMode = ! $scope.editMode;
           if ($scope.editMode){
             $scope.modelCopy = angular.copy($scope.adfModel, {});
+            $rootScope.$broadcast('adfDashboardEditMode', name, model);
           }
 
           if (!$scope.editMode){
             $rootScope.$broadcast('adfDashboardChanged', name, model);
           }
         };
+		
```

Comment:
Since we're paying so much attention to formatting now, I can't help but notice the indentation is off on this block.

Reply:
@david-oc-miller 

* I formatted the code 
* I ran gulp task "build" which calls tasks "css" & "js" (I believe these are the tasks which we need to run)
Review it and write me if i need to do something else, thanks.